docs/reference/api/connectivity/networksocket/MeshInterface.md

```diff
@@ -1,11 +1,92 @@
-<h2 id="mesh-api">Mesh</h2>
+<h2 id="mesh-api">6LoWPAN Mesh</h2>
 
-The Arm Mbed mesh API allows the application to use the IPv6 mesh network topologies through the [nanostack](/docs/development/reference/technology.html#nanostack) networking stack.
+#### 6LoWPAN Mesh Interfaces
+
+The Arm Mbed Mesh API allows the application to use the IPv6 mesh network topologies through the [Nanostack](/docs/v5.7/tutorials/mesh.html#nanostack) networking stack.
+
+Mbed OS provides two types of IPv6 based mesh networks:
+
+- 6LoWPAN_ND, loosely following the Zigbee-IP specification.
+- Thread, following the specification from Thread Group.
+
+Nanostack is the networking stack that provides both of these protocols. For more information on the stack internals, please refer to the [Nanostack documentation](/docs/v5.7/tutorials/mesh.html#nanostack). Application developers use Nanostack through the Mbed Mesh API.
+
+The application can use the `LoWPANNDInterface` or `ThreadInterface` object for connecting to the mesh network. When successfully connected, the application can use the Mbed C++ socket APIs to create a socket to start communication with a remote peer.
+
+Mbed OS provides `NanostackEthernetInterface` for Ethernet.
```

Comment:
This is still there to allow you to use any existing `NanostackEthernetPhy` drivers, but not really needed in the EMAC world. You should be using `EthernetInterface` to use EMAC drivers with Nanostack.

Maybe just drop the paragraph as it's nothing to do with mesh interfaces.

This does remind me that I should have at least actually stuck a "deprecated" on it. Main sticking point is that the K64F Nanostack driver still works a bit better than the EMAC one for cable detection logic.

Reply:
I'll drop the reference as it is deprecated.

I did not actually touch this part at all, just moved it away from Socket page to its proper location.